### PR TITLE
Highlight new sidebar items

### PR DESCRIPTION
**Pull request by fonurr**
- Projects or styleguides when added
- Screens or components when pinned

### CODE REVIEW

File: src/sidebar/pin/tree/PinTreeDataProvider.ts [hunk 1]
```diff
@@ -49,6 +51,22 @@ class PinTreeDataProvider extends TreeDataProvider {
...
+
+        this.reveal(screenTreeItem, `Screen: ${screen.name}|${screen._id}`);
+    }
+
+    public revealComponent(component: ResponseZeplinComponent) {
+        // Find component tree item
+        const componentTreeItem = this.getRoots()
+            .find(item => item instanceof ZeplinComponentTreeItem && item.zeplinComponent._id === component._id);
+
+        this.reveal(componentTreeItem, `Component: ${component.name}|${component._id}`);
```
Comment (konukselim) [inline src/sidebar/pin/tree/PinTreeDataProvider.ts:68]: shouldn't this be
```suggestion
        this.reveal(componentTreeItem, `Component: ${component.name || component._id}`);
```
or ts excepts yours as well?

Comment (fonurr) [reply src/sidebar/pin/tree/PinTreeDataProvider.ts:68]: this is not to log either of them. It is to log both of them in the format of `name|id`

**Review by konukselim — COMMENTED** [review]
Does highlight also scrolls to the item highlighted when items are overflowed and the item to be highlighted not shown in the area at the moment?

File: src/sidebar/pin/tree/PinTreeDataProvider.ts (hunk 1 above)
Comment (konukselim) [reply src/sidebar/pin/tree/PinTreeDataProvider.ts:68]: A-ha gotcha!

**Comment by fonurr** [comment]
We have no control over it, but yes, it does

**Review by mertkahyaoglu — APPROVED** [review]

**Review by konukselim — APPROVED** [review]

**Pull request merged**